src/sections/Community/Handbook/projects.js

```diff
@@ -16,19 +16,38 @@ const Maintainer= () => {
-            <h3>3. Service Mesh Performance</h3>
-            <p>The Service Mesh Performance (SMP) is a vendor-neutral specification for capturing details of environment and infrastructure details, service mesh and it’s configuration, service/application details, and bundling of statistical analysis of the result.</p>
-            <h3>4. GetNightHawk</h3>
-            <p>This Is a Layer 7 (HTTP/HTTPS/HTTP2) performance characterization tool.      Nighthawk is Envoy’s load generator and is written in C++. Meshery integrates Nighthawk as one of (currently) three choices of load generator for characterizing and managing the performance of service meshes and their workloads.</p>
+            <h3><a href="https://layer5.io/">1. Layer5</a></h3>
+            <p>Its service mesh community represents the largest collection of service Mesh projects.Emerging projects like GetNighthawk , community with  MeshMates, catch-all org, “home base”.<br/> Github link: <a href="https://github.com/layer5io">https://github.com/layer5io</a>
+              <ul>
+                <li><h4><a href="https://layer5.io/">Layer5 :</a></h4> </li>
+                <p>Layer5 is the official website of the Layer5 community showing an overview of the Layer5 projects. The different service mesh landscapes, resources to learn about service mesh, and communities.<br/> Github link: <a href="https://github.com/layer5io/layer5">https://github.com/layer5io/layer5</a> </p>
+                <li><h4><a href="https://layer5.io/projects/image-hub">Image-Hub :</a></h4> </li>
+                <p>Image Hub is a sample application written to run on Consul for exploring WebAssembly modules used as Envoy filters are written in Rust. These modules can be used to implement multi-tenancy or to implement per-user rate-limiting in your application's endpoints. This application was first demonstrated at DockerCon 2020.<br/> Github link: <a href="https://github.com/layer5io/image-hub">https://github.com/layer5io/image-hub</a> </p>
+                <li><h4><a href="https://layer5.io/projects/getnighthawk">GetNightHawk :</a></h4> </li>
+                <p>This Is a Layer 7 (HTTP/HTTPS/HTTP2) performance characterization tool. Nighthawk is Envoy’s load generator and is written in C++. Meshery integrates Nighthawk as one of (currently) three choices of load generators for characterizing and managing the performance of service meshes and their workloads.<br/> Github link: <a href="https://github.com/layer5io/getnighthawk">https://github.com/layer5io/getnighthawk</a> </p>
+              </ul>
+            </p>
+            <h3><a href="https://layer5.io/service-mesh-management/meshery">2. Meshery</a></h3>
```

Comment:
meshery logo here instead of `2.`  Similarly add logos for Service Mesh Performance and Service Mesh Patterns below.